docs/faq.md

```diff
@@ -0,0 +1,310 @@
+# Frequently Asked Questions
+
+### ABOUT THE PROJECT
+
+The goal of this project is to make a convenient and user-friendly GUI around Bitcoin Core with a focus on multisignature setup with airgapped (offline) hardware wallets. 
+
+We first wanted to make a new hardware wallet, but after we understood that everything can be hacked, we decided to build a user-friendly Multisig Desktop App and nice DIY Hardware wallet.
+
+**Why is that good for Bitcoin?**
+
+ - User: Better Security with multisig setup 
+ - User: Better Privacy with own node 
+ - HWW Makers: More HW wallets sold 
+ - Node Makers: More Nodes sold 
+ - Network: More nodes running 
+
+We can actually incentivize the Bitcoin community to run their own node with this user-friendly multisig & node setup! 
+
+*Why the name Specter?*
+
+We named it after a vulnerability since we learned that everything can be hacked! Although, when you combine different technical systems in an user-friendly multisig setup you can significantly increase the security of your cold storage. 
+
```

Comment:
Why the name Specter?
"A specter is haunting the modern world, the specter of crypto anarchy." 
> The Crypto Anarchist Manifesto - Timothy C. May - Sun, 22 Nov 92 12:11:24 PST 

Specter is that little ghost helping the sovereign cypherpunk to protect his property rights.
We are aware of the vulnerability (Spectre) and know there is an infinite game against vulnerabilities.
https://en.wikipedia.org/wiki/Spectre_(security_vulnerability)
In Bitcoin Cold storage we can use multisignature and different hardware wallets to mitigate these risks, while protecting our privacy by verifying transactions on our own node.